src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,7 +105,18 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
+
     driverController.b().onTrue(new CalibrateCmd(driveSub));
+
+    operatorController.povCenter().onTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleStart));
+    operatorController.povDown().onTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleFloor));
+    operatorController.povRight().onTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleProcessor));
+    operatorController.povLeft().onTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleL2));
+    operatorController.povUp().onTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleL3));
+
+    operatorController.a().onTrue(new IntakeSetVelocityPIDCmd(intakeSub, IntakeConstants.kMaxVelocity));
+    operatorController.b().onTrue(new IntakeStopCmd(intakeSub));
+    operatorController.y().onTrue(new ArmResetEncoderCmd(armSub));
```

Comment:
`onTrue` should only be used for commands that won't cause harm if it goes wrong. `whileTrue` would allow the operator to release the button in case of some fault (provided the `end` methods in the commands are implemented correctly, which they appear to be).

```suggestion
    operatorController.povCenter().whileTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleStart));
    operatorController.povDown().whileTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleFloor));
    operatorController.povRight().whileTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleProcessor));
    operatorController.povLeft().whileTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleL2));
    operatorController.povUp().whileTrue(new ArmSetPositionPIDCmd(armSub, ArmConstants.kAngleL3));

    operatorController.a().whileTrue(new IntakeSetVelocityPIDCmd(intakeSub, IntakeConstants.kMaxVelocity));
    operatorController.b().onTrue(new IntakeStopCmd(intakeSub));
    operatorController.y().onTrue(new ArmResetEncoderCmd(armSub));

Reply:
`ArmSetPositionPIDCmd` and `IntakeSetVelocityPIDCmd` are currently both instant commands, with the idea that the arm would continue to hold its position or intake would continue to hold its velocity until it's interrupted by another command. In this case, would `whileTrue` make any difference? 

I can change these two to regular commands so that the `end` methods would stop the arm/intake if you'd like.